StitchMDBKit/Podfile

```diff
@@ -0,0 +1,10 @@
+# Uncomment the next line to define a global platform for your project
```

Comment:
[request] Could we name this module something else? I'd rather it break a convention and be `LocalMongoDBService`. Is that bad?

Reply:
I think that's fine. I played around with the `Kit` naming because it is more "iOS"-y. But I think it's more important that we have unification throughout the SDKs.